templates/vm.xml.j2

```diff
@@ -35,13 +35,15 @@
       <driver name='qemu' type='{{ volume.format | default(libvirt_volume_default_format) }}'/>
       {% if volume.type | default(libvirt_volume_default_type) == 'file' %}
       <source file='{{ volume.file_path |default(libvirt_volume_default_images_path) }}/{{ volume.name}}'/>
+      {% elif volume.zfs_pool is defined and volume.zfs_pool is sameas true %}
+      <source dev='/dev/zvol/{{ ansible_local.libvirt.zfs_storage_pools[volume.pool] }}/{{ volume.name }}'/>
```

Comment:
This post suggests you can use the normal pool/volume configuration:

http://empt1e.blogspot.com/2014/08/zfs-support-in-libvirt.html

```
    <disk type='volume' device='disk'>
      <source pool='zfsfilepool' volume='vol1'/>
      <target dev='vdb' bus='virtio'/>
    </disk>
```

Reply:
I've updated the PR with this change